test/services/ValidatingPageBuilderSpec.scala

```diff
@@ -106,9 +107,34 @@ class ValidatingPageBuilderSpec extends BaseSpec with ProcessJson with StanzaHel
         case Left(List(PageOccursInMultiplSequenceFlows("5"), PageOccursInMultiplSequenceFlows("3"))) => succeed
         case Left(err) => fail(s"Attempt to parse page with unsupported page redirect failed with error ${err}")
       }
+    }
+
+    "Detect missing sequence flow termination caused by main flow connection into sequence flow" in new Test {
+      val flow: Map[String, Stanza] = Map(
+        Process.StartStanzaId -> PageStanza("/start", Seq("1"), stack = false),
+        "1" -> InstructionStanza(4, Seq("2"), None, stack = false),
+        "2" -> SequenceStanza(0, Seq("3", "5", "33"), Seq(2, 3), None, stack = false),
+        "3" -> PageStanza("/page-3", Seq("4"), stack = false),
+        "4" -> InstructionStanza(0, Seq("333"), None, stack = false),
+        "333" -> PageStanza("/page-333", Seq("444"), stack = false),
+        "444" -> InstructionStanza(0, Seq("8"), None, stack = false),
+        "33" -> PageStanza("/page-33", Seq("44"), stack = false),
+        "44" -> InstructionStanza(4, Seq("end"), None, stack = false),
+        "5" -> PageStanza("/page-5", Seq("6"), stack = false),
+        "6" -> InstructionStanza(0, Seq("end"), None, stack = false),
+        "8" -> InstructionStanza(0, Seq("end"), None, stack = false),
+        "end" -> EndStanza
+      )
+      val process = processWithLinks.copy(flow = flow)
 
+      pageBuilder.pagesWithValidation(process) match {
+        case Right(pages) => fail(s"Attempt to parse page with unsupported page redirect succeeded")
+        case Left(List(MissingUniqueFlowTerminator("3"))) => succeed
+        case Left(err) => fail(s"Attempt to parse page with unsupported page redirect failed with error ${err}")
+      }
```

Comment:
If you have time you could add a test where the main flow links into the sequence flow from the continuation after the sequence stanza.

Reply:
will keep in mind